src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -340,7 +269,8 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
   }
 
   it should "get list of clusters to auto freeze" in isolatedDbTest {
```

Comment:
I'm guessing the lines starting with `Cluster(` below got misaligned by IntelliJ?

Reply:
Yup, I think so. I was thinking this won't really make a difference in the end once they're all replaced (hopefully soon)